modules/brainbrowser/ajax/minc.php

```diff
@@ -22,13 +22,59 @@
 
 $headers = array();
 
-$query     = "select File from files where FileID = :MincID";
-$minc_file = $DB->pselectOne($query, array('MincID' => $_REQUEST['minc_id']));
-$minc_file = getMincLocation() . $minc_file;
+if (strpos($_REQUEST['minc_id'], 'l') !== false) {
```

Comment:
@johnsaigle, it's the small case letter "L" being checked here.  So it's not a number.  ;-)